mp/src/game/server/momentum/tickset.cpp

```diff
@@ -7,13 +7,32 @@
 // memdbgon must be the last include file in a .cpp file!!!
 #include "tier0/memdbgon.h"
 
-float* TickSet::interval_per_tick = nullptr;
+static void OnIntervalPerTickChange(IConVar *var, const char *pOldValue, float fOldValue);
```

Comment:
You can just define this method up here, there's really nothing special that it requires to have it be so far away (bottom of the file)